lib/modules/keyboardNav.js

```diff
@@ -789,7 +789,7 @@ module.options = {
 		title: 'keyboardNavsSubredditFrontPageTitle',
 		callback() {
 			const sub = currentSubreddit();
-			if (sub) navigateTo(false, `/r/${sub}`);
+			if (sub) navigateTo(`/r/${sub}`, { leaveNP: true });
```

Comment:
This is the only shortcut where I might consider setting `leaveNP` to `false`, since we're still browsing the same subreddit.